js/components/urlBar.js

```diff
@@ -368,6 +369,14 @@ class UrlBar extends ImmutableComponent {
       action='#'
       id='urlbar'
       ref='urlbar'>
+      <Button iconClass={this.titleMode ? 'fa-star' : 'fa-star-o'}
```

Comment:
Not sure I understand, why was bookmarks moved into the urlbar component? It seems unrelated to the rest of the changeset.